### PR TITLE
Fastroping - Fix inconsistent usage of ARR macro

### PR DESCRIPTION
**Pull request by Dahlgren**
**When merged this pull request will:**
- Change remaining conditions to use nested ARR macros like other statements in fastroping CfgVehicles
- Fix build of fastroping with newer mikero tools

### CODE REVIEW

**Comment by PabstMirror** [comment]
there are 4 commas, so ARR_5
otherwise it throws
```
File z\ace\addons\fastroping\CfgVehicles.hpp, line 210: '/CfgVehicles/Heli_Transport_02_base_F/UserActions/Ramp_Close.condition': Missing ';' at the end of line
Error context ])] call ace_fastroping_fnc_canCloseRamp);
ErrorMessage: File z\ace\addons\fastroping\CfgVehicles.hpp, line 210: '/CfgVehicles/Heli_Transport_02_base_F/UserActions/Ramp_Close.2': ']' encountered instead of '='
```

File: addons/fastroping/CfgVehicles.hpp [hunk 1]
```diff
@@ -221,7 +221,7 @@ class CfgVehicles {
         class UserActions {
             class Ramp_Open;
             class Ramp_Close: Ramp_Open {
-                condition = QUOTE([ARR_5(this,'CargoRamp_Open',[[0],[1],[2]])] call FUNC(canCloseRamp));
+                condition = QUOTE([ARR_3(this,'CargoRamp_Open',[[0],[1],[2]])] call FUNC(canCloseRamp));
```
Comment (commy2) [inline addons/fastroping/CfgVehicles.hpp:224]: This is wrong though. There are 4 commas, so 5 arguments. Macros don't operate in SQF syntax, it's a preprocessor of raw text.

**Comment by Dahlgren** [comment]
Changed the inner arrays to use ARR macros. Now it's compatible with both newer pboproject and cfgconvert.

**Comment by commy2** [comment]
Bad change. The current implementation better reflects what is happening during preprocessing. The new one is adding complexity and hides from the reader how the macros are resolved.

**Review by jonpas — APPROVED** [review]
This is fine for now if it assures compatibility with the tools (remember, we want ease of access, not more restrictions to building ACE).

Should also be reported to Mikero and this PR can be reverted if it ever gets fixed.

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**